source/guide_bludit.rst

```diff
@@ -0,0 +1,126 @@
+.. highlight:: console
+
+.. author:: Christian Kantelberg <uberlab@mailbox.org>
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/bludit.png
+      :align: center
+
+######
+Bludit
+######
+
+Bludit_ is a web application to build your own website or blog in seconds, it's completely free and open source. Bludit is a Flat-File CMS this means Bludit uses files in JSON format to store the content, you don't need to install or configure a database. You only need a web server with PHP support.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * PHP_
+  * Domains_
+
+License
+=======
+
+Bludit_ are released under the MIT License. 
+
+All relevant legal information can be found here 
+
+  * https://tldrlegal.com/license/mit-license
+  * https://docs.bludit.com/en/#license
+
+Prerequisites
+=============
+
+We're using PHP_ in the stable version 7.1:
+
+::
+
+ [isabell@stardust ~]$ uberspace tools version show php
+ Using 'PHP' version: '7.1'
+ [isabell@stardust ~]$
+
+Your blog domain needs to be setup:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+``cd`` to your `document root`_, then download and configure Bludit_.
+
+.. code-block:: console
+ :emphasize-lines: 1,2
+
+ [isabell@stardust ~]$ cd /var/www/virtual/$USER/html/
+ [isabell@stardust html]$ wget https://github.com/bludit/bludit/archive/3.8.0.tar.gz
+ […]
+ Saving to: ‘3.8.0.tar.gz’
+
+    [    <=>                                ] 1,016,833   1.58MB/s   in 0.6s
+
+ 2019-02-27 20:02:16 (1.58 MB/s) - ‘3.8.0.tar.gz’ saved [1016833]
+ [isabell@stardust html]$
+ 
+Untar the archive and then delete it. 
+
+.. code-block:: console
+ :emphasize-lines: 1,5
+ 
+ [isabell@stardust html]$ tar -xzvf 3.8.0.tar.gz --strip-components=1
+ bludit-3.8.0/.github/
+ […]
+ bludit-3.8.0/install.php
+ [isabell@stardust html]$ rm 3.8.0.tar.gz
+ [isabell@stardust html]$ 
+
+
+Finishing installation
+======================
+
+Now point your Browser to your installation URL ``https://isabell.uber.space/install.php``. 
+Complete the form and follow the installation instructions.
+
+You will need to enter the following information:
+
+  * language: the language you prefer.
+  * admin password: set up your admin password.
+  
+
+Tuning
+======
+
+For plugins, themes and other stuff go to Bludit_.
+
+
+Updates
```

Comment:
Can you please add the concrete commands used to do this? Especially the step preserving `/bl-content/database/` would be very helpful to users!

Reply:
You mean the update process?
Mhm, since the update from the client to the server (see list) is performed, the way is actually already shown. For example, I use Transmit for such things. Other use e.g. Filezilla or the terminal.
I will try to write some textlines for using the terminal.

Reply:
Just like installation, the update should also be possible via the command line, to keep things consistent. Additionally, we don't require users to have "special" software like an sftp client on their machines.

Reply:
Done 
- add update process with terminal
- add folder structure with focus to important area
- add password recovery for admin account